sstables/entry_descriptor.hh

```diff
@@ -21,6 +21,8 @@
 
 #pragma once
```

Comment:
What's the motivation? You're declaring static file/dir types, but I can't guess why.

Reply:
We're practically using `pragma once` as an include guard.
Isn't that by convention?